app/Models/Product.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\SoftDeletes;
+
+
+class Product extends Model
+{
+    use HasFactory, SoftDeletes;
+
+
+    protected $table = ('products');
+    protected $fillable = [
+        'ProductName',
```

Comment:
`ProductName` is not the same as `productName` 

you have mistaken the cases

Reply:
i have opened your migration file and find it `productName` so your mistake in the model here change the case